airflow/dag_processing/manager.py

```diff
@@ -1216,7 +1216,7 @@ def _kill_timed_out_processors(self):
                 Stats.decr("dag_processing.processes", tags={"file_path": file_path, "action": "timeout"})
                 Stats.incr("dag_processing.processor_timeouts", tags={"file_path": file_path})
                 # Deprecated; may be removed in a future Airflow release.
-                Stats.incr("dag_file_processor_timeouts")
+                Stats.incr("dag_file_processor_timeouts", tags={"file_path": file_path})
```

Comment:
I’m not sure tbis is a good idea; we want to nudge people toward the new stat instead.

Reply:
How so?  Stats is an interface that both OTel and StatsD will share.

Reply:
Yes only this one stat

Reply:
Sure, that makes sense.   I'll remove the tagging on that one.  I'm out of town on holiday for a few days, I'll get that and the tests fixed on Tuesday when I'm home.

Reply:
Reverted this in https://github.com/apache/airflow/pull/30496/commits/3c32c7df8d17fa685011f7f6a39b40fee944315b